library/common/http/client.h

```diff
@@ -69,6 +69,13 @@ class Client : public Logger::Loggable<Logger::Id::http> {
    */
   void sendHeaders(envoy_stream_t stream, envoy_headers headers, bool end_stream);
 
+  /**
+   * Notify the stream that the caller is ready to receive more data. Only used in explicit
+   * buffering mode.
+   * @param bytes_to_read, the quantity of data the caller is prepared to process.
+   */
+  void readData(envoy_stream_t /*stream*/, size_t /*bytes_to_read*/) {}
```

Comment:
maybe clarify if this is request or response data for all of these, since we'll eventually have calls for both?